web/api/wrapper.go

```diff
@@ -46,6 +47,11 @@ func (wc WrapperCommand) Execute() (output string, err error) {
 
 	// Execute command and capture output
 	command := exec.Command("./binaries/oauth2l", args...)
+
+	// If there is a code needed to be inputted, the code will be inputted.
```

Comment:
I just tried this out with the Vercel preview and got a 400 bad request response in the token output field after approving the user OAuth and entering the code. Looking at your PR top-level comment, it sounds like you will try to run the oauth2l command twice, in which the case the first run will fail on purpose? Can you think of a way to only run oauth2l once? Such as blocking on input in a separate thread?

Reply:
I think the reason why is because the vercel is using the deployed backend, which has not been updated yet with the new backend that handles the client id part.

Reply:
Gotcha.